engine/common/src/main/java/org/enso/common/Platform.java

```diff
@@ -48,6 +35,19 @@ public static Platform getOperatingSystem() {
     return OPERATING_SYSTEM;
   }
 
+  /**
+   * Has the same behavior as {@link java.io.File#separatorChar}. Use this method if you want to
+   * avoid initializing {@link java.io.File} class.
+   *
+   * @return the file separator character for the current operating system.
+   */
+  public static char separatorChar() {
```

Comment:
- nice to see `separatorChar` method
- encapsulates the logic nicely
- when I was dreaming about it
- I was dreaming about **instance** method, not `static` one...